docs/sources/reference/components/prometheus/prometheus.exporter.ssh.md

```diff
@@ -0,0 +1,205 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/prometheus/prometheus.exporter.ssh/
+aliases:
+  - ../prometheus.exporter.ssh/ # /docs/alloy/latest/reference/components/prometheus.exporter.ssh/
+description: Learn about prometheus.exporter.ssh
+title: prometheus.exporter.ssh
+---
+
+# prometheus.exporter.ssh
+
+The `prometheus.exporter.ssh` component embeds an SSH exporter for collecting metrics from remote servers over SSH and exporting them as Prometheus metrics.
+
+## Usage
+
+```
+prometheus.exporter.ssh "LABEL" {
+  // Configuration options
+}
+```
+
+## Arguments
+
+The following arguments can be used to configure the exporter's behavior.
+All arguments are optional unless specified. Omitted fields take their default values.
```

Comment:
Observation: In reviewing this I've noticed that the wording of some sections in the Prometheus topics are not consistent with the rest of the components. For example, Arguments in other sections simply state: `The following arguments are supported:`

I'll create an Issue to follow up on this.  No action or suggestions for this PR.